build/scripts/build-ccutil.sh

```diff
@@ -70,3 +70,6 @@ else
       echo "<li><a href=./${BRANCH}/index.html>${BRANCH}</a></li>" >> titles-generated/index.html
   fi
 fi
+
+# Test the links with htmltest
+podman run --interactive --rm --tty --volume "$(pwd)":/test:Z docker.io/wjdp/htmltest:latest -c .htmltest.yml
```

Comment:
Can you really run this in `interactive` mode as part of a non-interactive GH action?

```suggestion
podman run --rm --tty --volume "$(pwd)":/test:Z docker.io/wjdp/htmltest:latest -c .htmltest.yml
```

Will a failure here cause the PR check to fail? do you need to return some error code, or will the `docker.io/wjdp/htmltest:latest` exit with an error if links are broken?

Reply:
It exits with an error:

```
$ podman run --rm --tty   --volume "$(pwd)":/test:Z   docker.io/wjdp/htmltest:latest -c .htmltest.yml 
htmltest started at 06:40:45 on titles-generated
========================================================================
broken.html
  Get "http://nowhere.real/broken": dial tcp: lookup nowhere.real on 169.254.1.1:53: no such host --- broken.html --> http://nowhere.real/broken
========================================================================                                                                                                       
✘✘✘ failed in 45.773955ms                                                                                                                                                      
1 errors in 11 documents                                                                                                                                                       
$ echo $?
1
```